devicedefender/CMakeLists.txt

```diff
@@ -111,6 +114,12 @@ else()
     set(TARGET_DIR "static")
 endif()
 
+include(CMakePackageConfigHelpers)
+write_basic_package_version_file(
+        "${CMAKE_CURRENT_BINARY_DIR}/iotdevicedefender-cpp-config-version.cmake"
```

Comment:
The reason I'm using a hard-coded name here, instead of something like `${PROJECT_NAME}` is because the project name is `IotDeviceDefender` while the normal Cmake file is hard-coded to `iotdevicedefender`, so to keep it consistent, this is also hard coded with a similar naming scheme.